base/src/main/java/com/nedap/archie/base/MultiplicityInterval.java

```diff
@@ -109,13 +109,28 @@ public boolean upperIsOne() {
         return has(1) && !has(2);
     }
 
+    @Override
     public String toString() {
-        if(isOpen()) {
-            return getLower() + MULTIPLICITY_RANGE_MARKER + MULTIPLICITY_UNBOUNDED_MARKER;
+        Integer lower = getLower();
+        Integer upper = getUpper();
+        StringBuilder result = new StringBuilder();
+        if (isLowerUnbounded()) {
+            result.append(MULTIPLICITY_UNBOUNDED_MARKER);
```

Comment:
This is not something one would do with a multiplicity interval (lower cannot be less than 0 I would say), but good to support it I guess.